aten/src/THC/THCAllocator.cpp

```diff
@@ -4,21 +4,30 @@ THCIpcDeleter::~THCIpcDeleter() {
   int prev_device;
   THCudaCheck(cudaGetDevice(&prev_device));
   THCudaCheck(cudaSetDevice(device_));
-  THCudaCheck(cudaIpcCloseMemHandle(data_));
   THCudaCheck(cudaSetDevice(prev_device));
 }
 
 void deleteTHCIpcDeleter(void* ptr) {
   delete static_cast<THCIpcDeleter*>(ptr);
 }
 
-at::DataPtr THCIpcDeleter::makeDataPtr(void* data, int device) {
+// Refer to NB [CUDA IPC and the caching allocator] for more details
+// basePtr - device ptr of allocated CUDA memory region. This memory region
+//           might contain storages of different types.
+// data    - ptr to where the storage (of a single type) should start.
```

Comment:
nit: I find it's always useful to state invariants when I'm documenting things. In this case, the invariant is that `data` must lie within the CUDA memory allocation represented by `basePtr`.